frontend_lib/src/appContentFactory.js

```diff
@@ -678,24 +636,17 @@ export function appContentFactory (WrappedComponent) {
       )
     }
 
-    appContentSaveNewComment = async (
-      content, isCommentWysiwyg, newComment, newCommentAsFileList, setState, appSlug, loggedUsername, id = ''
-    ) => {
+    appContentSaveNewCommentFileList = async (setState, content, fileList) => {
       this.checkApiUrl()
-      if (newComment) {
-        await this.saveCommentAsText(content, isCommentWysiwyg, newComment, setState, appSlug, loggedUsername, id)
-      }
 
-      if (newCommentAsFileList && newCommentAsFileList.length > 0) {
-        setState({ isFileCommentLoading: true })
-        const responseList = await Promise.all(
-          newCommentAsFileList.map(newCommentAsFile => this.saveCommentAsFile(content, newCommentAsFile))
-        )
-        const uploadFailedList = responseList.filter(oneUpload => isFileUploadInErrorState(oneUpload))
-        uploadFailedList.forEach(fileInError => sendGlobalFlashMessage(fileInError.errorMessage))
+      setState({ isFileCommentLoading: true })
+      const responseList = await Promise.all(
+        fileList.map(file => this.appContentSaveNewCommentFile(content, file))
+      )
+      const uploadFailedList = responseList.filter(upload => isFileUploadInErrorState(upload))
+      uploadFailedList.forEach(fileInError => sendGlobalFlashMessage(fileInError.errorMessage))
```

Comment:
I think globalFlashMessage cannot appear more than once. So it should be better to merge all message into one globalFlashMessage